Source/Core/VideoBackends/OGL/OGLTexture.cpp

```diff
@@ -0,0 +1,241 @@
+#include "Common/GL/GLInterfaceBase.h"
```

Comment:
What's this for?  It doesn't include the necessary GLInterface needed to compile.